locales/ko.yaml

```diff
@@ -6,111 +6,111 @@ page-title:
   home: 홈
   song: 곡 정보
   songs: 곡 목록
-  gallery: 단위인정
-  about: 사이트 정보
-  bug-report: 문의
+  gallery: 트랙 목록
+  about: 정보
+  bug-report: 문제 보고
   source-code: 소스 코드
 
 # General UIs
 ui:
-  ok: OK
+  ok: 확인
   cancel: 취소
   close: 닫기
   search: 검색
-  goBack: 뒤로
-  all: 전체
+  goBack: 뒤로 가기
+  all: 모두
   default: 기본값
   sort-asc: 오름차순
   sort-desc: 내림차순
 
 # Terminologies
 term:
-  category: 카테고리
+  category: 범주
   title: 제목
-  artist: 아티스트명
+  artist: 아티스트
   version: 버전
   releaseDate: 수록일
   bpm: BPM
   minBPM: 최소 BPM
   maxBPM: 최대 BPM
-  sheets: 보면
-  type: 타입
+  sheets: 악보
+  type: 유형
   difficulty: 난이도
-  level: 레벨
-  minLevel: 최소 레벨
-  maxLevel: 최대 레벨
-  internalLevel: 보면상수
-  noteDesigner: 보면 제작자
+  level: 수준
```

Comment:
This change changes the literal word 'level' to its korean translated word;

I think this is better off not translated since the word 'level' is dominantly used in a gaming context